src/routes/router.ts

```diff
@@ -15,28 +14,25 @@ router.use(async (ctx, next) => {
     }
 });
 
-router.get('/products', async ctx => {
+router.get('/products', async (ctx) => {
```

Comment:
pass ctx to the controller

```suggestion
router.get('/products', productController.getProducts);
```

Reply:
why?

Reply:
standard, and more clear.
if your controller will set 2 different statuses in different cases

Reply:
by the way, I didn't found any assignment to status codes (201 for created for example)